src/constants/actions/index.js

```diff
@@ -10,10 +10,15 @@ export const NEW_KEYS = 'NEW_KEYS';
 export const COMPLETE_SWAP = 'COMPLETE_SWAP';
 export const SET_SWAP_AMOUNT = 'SET_SWAP_AMOUNT';
 export const SET_SWAP_INVOICE = 'SET_SWAP_INVOICE';
-export const SWAP_REQUEST = 'REQUEST_SWAP';
+export const SWAP_REQUEST = 'SWAP_REQUEST';
 export const SWAP_RESPONSE = 'SWAP_RESPONSE';
 
 /**
  * Refund actions
  */
 export const ENTER_REFUND_MODE = 'ENTER_REFUND_MODE';
```

Comment:
We can remove 'ENTER_REFUND_MODE` action, instead use an action such as `COMPLETE_SWAP` that clears all the refund state when exiting a refund or completing one